utils/browser_helper.py

```diff
@@ -0,0 +1,53 @@
+from loguru import logger
+from selene import be
+from selenium.common.exceptions import NoSuchElementException, ElementNotVisibleException, \
+    ElementNotInteractableException, TimeoutException
+
+
+class ElementInteractions:
+    def __init__(self, browser):
+        self.browser = browser
+
```

Comment:
Should we initialize instance of ElementInteractions class or stay out of it (remove init method and self from every method below?

Reply:
### Closed. 
We decided to create instance factory.